presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -155,6 +157,10 @@
     private boolean adaptiveFilterReorderingEnabled = true;
     private boolean zstdJniDecompressionEnabled;
 
+    private Duration fileStatusCacheExpireAfterWrite = new Duration(1, TimeUnit.MINUTES);
```

Comment:
Maybe we should make the default value to be unused? For example, expire immediately after write and keep cache size to be 0. Though I understand that by setting `fileStatusCacheTables` to empty list, this feature is effectively disabled. Open to different opinions :)

Reply:
good point. let me update it expire in 0 seconds